brainy-ideas/readAloud-valence-jess/readme.md

```diff
@@ -0,0 +1,145 @@
+* **A**: positive activation and negative compensation: fewer disfluencies in negative>positive switches than reverse
+* **B**: general surprisal: similar disfluency rates positive>negative and negative>positive
+* **C**: mood-incongruent surprisal: enhanced disfluency rates when hitting switch that conflicts with current mood state
+
+    **Independent Variable**
+    - variable 1: type of switch (pos>neg or neg>pos) {categorical}
+    - variables 2: participant measures on initial questionnaires (BMIS, etc.) {continuous}
+
+    **Dependent Variable**<br/>
+    percentage of disfluent syllables across three factors:
+
+    | pre-switch group| switch group | post-switch group|
+    |:-: | :-: | :-: |
+    | 7 - 6 - 5 - 4 - 3 | 2 - 1 - switch - 1 - 2 | 3 - 4 - 5 - 6 - 7 |
+
+    1. within the switch group (two words prior to the switch, the switch itself, and two words following the switch)
+    2. in the word group prior to the switch (the five words prior to the switch group)
+    3. in the post-switch word group (the five words that following the switch group)
+
+    *Note 1*: I feel that syllables form a logical 'unit' in which to measure disfluency.  However, as semantic integration is required to prepare articulation during reading aloud, I propose that we slice on the word level for analyzing disfluency around the switch.<br/>
+    *Note 2*: As a starting point, I selected groups of five words; this may need to be revised.  Some prior research has looked into how far ahead people begin semantic and phonological encoding (so they understand an upcoming word and prepare to articulate it). For instance, if a participant is articulating three words prior to the switch word, but they have already seen the switch word and begun to integrate it semantically (and assuming this does actually affect their performance in reading aloud), they might stumble three words before the actual switch word. Alternately, they might recruit all the necessary resources to articulate the switch correctly...and then stumble after they successfully make it past the switch word. My to-read list includes [this article](https://www.frontiersin.org/articles/10.3389/fpsyg.2015.01432/full), which may provide better guidance.
+
+**Design:**
+
+Prior to the experimental task, participants complete several questionnaires:
+- Demographic questionnaire
+- Assessment of reading skill (TBD)
+- Brief Mood Introspection Scale (BMIS)
+- Emotion Regulation Questionnaire (ERQ)
+- Interpersonal Emotion Regulation Questionnaire (IERQ)
+- Beck Depression Inventory (BDI-II)
```

Comment:
but only if it is free, i think it might not be...

Reply:
How can we tell?  Pearson has a paid version, but there are plenty of free versions online.  Not sure what counts as "public domain" here...

Reply:
people post copies all the time online. but you have to check the license to see if you are actually supposed to use it or not. There are plenty of alternatives to BDI anyways, E.g., PHQ